File: moreapps/src/main/java/io/github/raghavsatyadev/moreapps/listener/MoreAppsDialogListener.java
package io.github.raghavsatyadev.moreapps.listener;

import io.github.raghavsatyadev.moreapps.model.MoreAppsDetails;

public interface MoreAppsDialogListener {
    void onClose();

    void onAppClicked(MoreAppsDetails appsModel);
}

File: moreapps/src/main/java/io/github/raghavsatyadev/moreapps/listener/MoreAppsDialogListener.kt
package io.github.raghavsatyadev.moreapps.listener

import io.github.raghavsatyadev.moreapps.model.MoreAppsDetails

interface MoreAppsDialogListener {
    fun onClose()
    fun onAppClicked(appsModel: MoreAppsDetails)
}